src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -1370,6 +1370,8 @@ void AllocateSurfaceHeatBalArrays(EnergyPlusData &state)
 
     state.dataHeatBalSurf->SurfQRadLightsInReport.dimension(state.dataSurface->TotSurfaces, 0.0);
     state.dataHeatBalSurf->SurfQdotRadLightsInRep.dimension(state.dataSurface->TotSurfaces, 0.0);
+    state.dataHeatBalSurf->SurfQRadLightsInReportOtherZones.dimension(state.dataSurface->TotSurfaces, 0.0);
```

Comment:
More of this `InRep` and `InReport` stuff.

Reply:
I have fixed all of these in the latest commit that just got pushed.  Everything is now InRep for consistency.  I think in some cases the name was shortened so the variable names were not "too long" perhaps.  Or maybe it was different developers.  Whatever the case, this should be resolved now.